.github/workflows/reusable-performance.yml

```diff
@@ -163,9 +167,21 @@ jobs:
           docker compose run --rm php php -i
           docker compose run --rm php locale -a
 
+      - name: Install object cache drop-in
+        if: ${{ inputs.memcached }}
+        run: cp src/wp-content/object-cache.php build/wp-content/object-cache.php
```

Comment:
Just curious, why did you move this here from above?

Reply:
Probably by accident when resolving merge conflicts.

Reply:
Let's move it back, if there's no good reason that it was moved? Probably works here too, but better to not change it if it worked before.